src/soroban/api.ts

```diff
@@ -264,6 +277,9 @@ export namespace Api {
 
     /** present only for invocation simulation */
     result?: SimulateHostFunctionResult;
+
+    // State Difference information
```

Comment:
please use jsdoc format for all of the fields like in the rest of this file so that they render correctly!

either

```
/**
 * multi line
 * comment
 */
```

or 

```
/** one liner */
```